manifests/c/com/pizi/scrcpygui/1.4.0/com.pizi.scrcpygui.yaml

```
@@ -0,0 +1,8 @@
+# Created with komac v2.13.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.10.0.schema.json
+
+PackageIdentifier: com.pizi.scrcpygui

```

```suggestion
PackageIdentifier: pizi.scrcpygui
```
Please update file paths to match this change.